docs/source/extension/extension_migration.rst

```diff
@@ -88,6 +88,22 @@ is to force packages deduplication using:
 
    jlpm dedupe --strategy highest
 
+Extension Status
+^^^^^^^^^^^^^^^^
+The extension has four possible status reportings: red 'X', red 'DISABLED', green 'OK', and green 'ENABLED'.
+The extension status is determined in the following ways:
+- Red 'X': the extension is reporting compatibility errors
+- Red 'DISABLED': the extension is disabled.
+- Green 'OK': the extension is installed.
+- Green 'ENABLED': the extension is installed and enabled.
+
+When upgrading extensions manually you may encounter the red 'X' status that indicates compatibility errors
+in the dependencies of your extension. It is likely that there are dependencies, in addition those which
+are JupyterLab packages, that are being compared to the dependencies of core JupyterLab and whose versions
+are not coinciding. Some of those dependencies may include the following: yjs, react, react-dom, typescript,
```

Comment:
did you encounter an error with typescript? If so could you please provide an error trace as this should never happen for development dependencies (like typescript) - only runtime dependencies may be incompatible.

```suggestion
are not coinciding. Some of those dependencies may include the following: yjs, react, react-dom,
```